### DISCUSSION TITLE
Complex Regression using GPytorch ?

### DISCUSSION THREAD

**Discussion by louisreine** [cornellius-gp/gpytorch#2601] [General]
Hello ! 

I'd like to know if GPytorch is compatible with complex valued data ? Sorry if I missed something, I'm a new user of Gaussian Processes in general. It maybe be a straightforward yes in the case of proper complex gaussian process, but otherwise I believe no ? 

I've seen that paper that translate the gaussian process for complex valued data that explains it fairly well : 10.1109/TNNLS.2018.2805019

Thank you in advance !

**Comment by gpleiss** [comment]
GPyTorch doesn't currently support complex-valued data. We don't have any plans to support it in the near future. (It's not something we're opposed to; we just don't have the time to add this feature.)

**Comment by louisreine** [comment]
Ok, thanks for the clear statement !
